### ISSUE TITLE
Install distro, then exit program and distro does not appear on the list. Various distros

### ISSUE THREAD

**Issue by miglrodri** [mbusb/multibootusb#269]
Hello,

Windows 10.
Version: Multiboot v9.1.0
Distros tried:

- neon-useredition-20180207-1523-amd64.iso
- Mageia-6-LiveDVD-Plasma-x86_64-DVD.iso
- kali-linux-2018.1-i386.iso
- elementaryos-0.4.1-stable.20180214.iso
- slax-32bit-9.3.0.iso
- tails-amd64-3.5.iso

Formatted the pen, tried installing again. Not a single distro gets installed and appears on the list.

Is this issue related with https://github.com/mbusb/multibootusb/issues/256

How can i fix it? When you release new version as installer.

Best regards,
Miguel.

Log: 

> 19:30:34.775 root INFO Running multibootusb version 9.1.0 from installed system...
19:30:34.833 root INFO Starting multibootusb GUI...
19:30:34.988 root INFO Cleaning old multibootusb directory...
19:30:34.996 root INFO Extracting syslinux to multibootusb directory...
19:30:35.069 root INFO Extracting syslinux modules to multibootusb directory...
19:30:35.828 root INFO Selected device E:
19:30:36.469 root INFO b'Name                   Type                        \r\r\nDisco #1, Parti\x87\xc6o #0  Extended w/Extended Int 13  \r\r\nDisco #0, Parti\x87\xc6o #0  Installable File System     \r\r\nDisco #0, Parti\x87\xc6o #1  Installable File System     \r\r\nDisco #0, Parti\x87\xc6o #2  Unknown                     \r\r\nDisco #2, Parti\x87\xc6o #0  Unknown                     \r\r\n\r\r\n'
19:33:53.164 root INFO No USB disk found...
19:33:53.169 root INFO Selected device E:
19:33:53.734 root INFO b'Name                   Type                        \r\r\nDisco #1, Parti\x87\xc6o #0  Extended w/Extended Int 13  \r\r\nDisco #0, Parti\x87\xc6o #0  Installable File System     \r\r\nDisco #0, Parti\x87\xc6o #1  Installable File System     \r\r\nDisco #0, Parti\x87\xc6o #2  Unknown                     \r\r\nDisco #2, Parti\x87\xc6o #0  Unknown                     \r\r\n\r\r\n'
19:34:47.246 root INFO Running multibootusb version 9.1.0 from installed system...
19:34:47.296 root INFO Starting multibootusb GUI...
19:34:47.338 root INFO Cleaning old multibootusb directory...
19:34:47.342 root INFO Extracting syslinux to multibootusb directory...
19:34:47.418 root INFO Extracting syslinux modules to multibootusb directory...
19:34:48.228 root INFO Selected device E:
19:34:48.799 root INFO b'Name                   Type                        \r\r\nDisco #1, Parti\x87\xc6o #0  Extended w/Extended Int 13  \r\r\nDisco #0, Parti\x87\xc6o #0  Installable File System     \r\r\nDisco #0, Parti\x87\xc6o #1  Installable File System     \r\r\nDisco #0, Parti\x87\xc6o #2  Unknown                     \r\r\nDisco #2, Parti\x87\xc6o #0  Unknown                     \r\r\n\r\r\n'
19:35:03.652 root INFO Executing ==> "C:\Program Files (x86)\multibootusb\data\tools\7zip\7z.exe" -ssc- -bb1 x -y "C:\Users\migue\Downloads\neon-useredition-20180207-1523-amd64.iso" -oC:\Users\migue\AppData\Local\Temp\multibootusb\iso_cfg_ext_dir "*.cfg" -r
19:35:03.813 root INFO Executing ==> "C:\Program Files (x86)\multibootusb\data\tools\7zip\7z.exe" -ssc- -bb1 x -y "C:\Users\migue\Downloads\neon-useredition-20180207-1523-amd64.iso" -oC:\Users\migue\AppData\Local\Temp\multibootusb\iso_cfg_ext_dir "*.txt" -r
19:35:03.934 root INFO Executing ==> "C:\Program Files (x86)\multibootusb\data\tools\7zip\7z.exe" -ssc- -bb1 x -y "C:\Users\migue\Downloads\neon-useredition-20180207-1523-amd64.iso" -oC:\Users\migue\AppData\Local\Temp\multibootusb\iso_cfg_ext_dir "*isolinux.bin" -r
19:35:04.019 root INFO Executing ==> "C:\Program Files (x86)\multibootusb\data\tools\7zip\7z.exe" -ssc- -bb1 x -y "C:\Users\migue\Downloads\neon-useredition-20180207-1523-amd64.iso" -oC:\Users\migue\AppData\Local\Temp\multibootusb\iso_cfg_ext_dir "*.lst" -r
19:35:04.190 root INFO Persistence option is available.
19:35:04.194 root INFO Persistence Max Size: 4.0G
19:35:18.848 root INFO MultiBoot Install: USB Disk: E:
19:35:18.852 root INFO MultiBoot Install: USB Label: KINGSTON
19:35:18.856 root INFO MultiBoot Install: USB UUID: 6856-D6AE
19:35:18.859 root INFO MultiBoot Install: USB mount path: E:\
19:35:18.864 root INFO MultiBoot Install: Disk total size: 28.9G
19:35:18.868 root INFO MultiBoot Install: Disk used size: 128.0K
19:35:18.872 root INFO MultiBoot Install: Disk free size: 28.9G
19:35:18.876 root INFO MultiBoot Install: Filesystem: FAT32
19:35:18.880 root INFO MultiBoot Install: Disk vendor: Not_Found
19:35:18.884 root INFO MultiBoot Install: Disk model: Not_Found
19:35:18.888 root INFO MultiBoot Install: ISO file: neon-useredition-20180207-1523-amd64.iso
19:35:18.892 root INFO MultiBoot Install: Distro type detected: ubuntu
19:35:18.896 root INFO Persistence chosen is 452.0M
19:35:20.086 root INFO Copying multibootusb directory to E:\
19:35:43.521 root INFO EFI/BOOT directory does not exist. Creating new.
19:35:43.591 root INFO Copying EFI directory to E:\
19:35:44.055 root INFO Installing neon-useredition-20180207-1523-amd64.iso on E:\multibootusb\neon-useredition-20180207-1523-amd64
19:35:44.060 root INFO Executing ==> "C:\Program Files (x86)\multibootusb\data\tools\7zip\7z.exe" -ssc- -bb1 x -y -o"E:\multibootusb\neon-useredition-20180207-1523-amd64" "C:\Users\migue\Downloads\neon-useredition-20180207-1523-amd64.iso"
19:36:46.346 root INFO Creating persistence...
19:36:46.351 root INFO Executing ==>"C:\Program Files (x86)\multibootusb\data\tools\dd\dd.exe" if=/dev/zero of="E:\multibootusb\neon-useredition-20180207-1523-amd64\casper-rw" bs=1M count=452
19:36:59.809 root INFO 
Successfully created persistence file...
19:36:59.813 root INFO Applying filesystem to persistence file...
19:36:59.817 root INFO Executing ==> echo y|"C:\Program Files (x86)\multibootusb\data\tools\mkfs\mke2fs.exe" -b 1024 -L casper-rw "E:\multibootusb\neon-useredition-20180207-1523-amd64\casper-rw"
19:37:34.393 root INFO 
Successfully applied filesystem...
19:37:34.557 root INFO Distro extraction completed...
19:37:34.932 root INFO 
Found syslinux version 6
19:37:34.936 root INFO 
Executing ==> C:\Users\migue\AppData\Local\Temp\multibootusb\syslinux\bin\syslinux6.exe -i -d /multibootusb/neon-useredition-20180207-1523-amd64/isolinux E: E:\multibootusb\neon-useredition-20180207-1523-amd64\isolinux\ubuntu.bs
19:37:35.353 root INFO 
Syslinux install was successful on distro directory...

**Comment by mbusb** [comment]
I can see where the issue lies. You are running a system which is not English. This has been fixed in the latest devel branch. Will release the latest version soon.

**Comment by nuk380y** [comment]
Sorry for commenting on a closed ticket, but I'm seeing this issue in the latest compiled RPM for Fedora (version 9.2.0).  It could be that I haven't upgraded to F31 from F30, but I'm not sure.

Anyway, the issue I'm seeing is that once I click on the "Install Distro" button, the window closes and there is no indication of the ISO being loaded to my USB drive.  I've tried the latest versions (as of this writing) for ArcoLinux, CentOS, Manjaro, and Pop!_OS.

Log for attempts:
```19:21:06.930 root INFO Could not find resource 'data/tools/gdisk/gdisk.exe'.
19:21:07.001 root INFO Using python version 3.7.5 (default, Oct 17 2019, 12:09:47) 
[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)]on platform Linux-5.3.16-200.fc30.x86_64-x86_64-with-fedora-30-Thirty
19:21:07.001 root INFO Running multibootusb version 9.2.0 from installed system...
19:21:07.008 root INFO Starting multibootusb GUI...
19:21:07.097 root INFO Extracting syslinux to multibootusb directory...
19:21:07.097 root INFO Host OS is 64 bit...
19:21:07.097 root INFO Extracting syslinux 64 bit...
19:21:07.107 root INFO Extracting syslinux modules to multibootusb directory...
19:21:07.173 root INFO Using pyudev for detecting USB drives...
19:21:07.186 root INFO 	/dev/sda
19:21:07.186 root INFO 	/dev/sda1
19:21:07.188 root INFO Selected device /dev/sda
19:21:07.950 root INFO Device /dev/sda is a MBR disk...
19:21:08.068 root INFO USB disk is not mounted and can't update list widget...
19:21:16.574 root INFO No USB disk found...
19:21:16.575 root INFO Using pyudev for detecting USB drives...
19:21:16.595 root INFO 	/dev/sda
19:21:16.596 root INFO 	/dev/sda1
19:21:16.605 root INFO Selected device /dev/sda
19:21:17.194 root INFO Device /dev/sda is a MBR disk...
19:21:17.345 root INFO USB disk is not mounted and can't update list widget...
19:21:38.567 root INFO Selected device /dev/sda1
19:22:10.127 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/arcolinux-v19.12.17.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.cfg" -r 2> /dev/null
19:22:10.161 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/arcolinux-v19.12.17.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.txt" -r 2> /dev/null
19:22:10.190 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/arcolinux-v19.12.17.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*isolinux.bin" -r 2> /dev/null
19:22:10.218 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/arcolinux-v19.12.17.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.lst" -r 2> /dev/null
19:22:10.390 root INFO Persistence support is not available for arcolinux-v19.12.17.iso
19:23:15.387 root INFO Could not find resource 'data/tools/gdisk/gdisk.exe'.
19:23:15.426 root INFO Using python version 3.7.5 (default, Oct 17 2019, 12:09:47) 
[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)]on platform Linux-5.3.16-200.fc30.x86_64-x86_64-with-fedora-30-Thirty
19:23:15.427 root INFO Running multibootusb version 9.2.0 from installed system...
19:23:15.429 root INFO Starting multibootusb GUI...
19:23:15.487 root INFO Cleaning old multibootusb directory...
19:23:15.488 root INFO Syslinux exist in multibootusb directory...
19:23:15.488 root INFO Using pyudev for detecting USB drives...
19:23:15.503 root INFO 	/dev/sda
19:23:15.503 root INFO 	/dev/sda1
19:23:15.506 root INFO Selected device /dev/sda
19:23:16.184 root INFO Device /dev/sda is a MBR disk...
19:23:16.303 root INFO USB disk is not mounted and can't update list widget...
19:23:20.219 root INFO Selected device /dev/sda1
19:23:48.286 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/CentOS-8-x86_64-1905-dvd1.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.cfg" -r 2> /dev/null
19:23:48.368 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/CentOS-8-x86_64-1905-dvd1.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.txt" -r 2> /dev/null
19:23:48.445 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/CentOS-8-x86_64-1905-dvd1.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*isolinux.bin" -r 2> /dev/null
19:23:48.524 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/CentOS-8-x86_64-1905-dvd1.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.lst" -r 2> /dev/null
19:23:48.758 root INFO Persistence support is not available for CentOS-8-x86_64-1905-dvd1.iso
19:26:05.082 root INFO Could not find resource 'data/tools/gdisk/gdisk.exe'.
19:26:05.122 root INFO Using python version 3.7.5 (default, Oct 17 2019, 12:09:47) 
[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)]on platform Linux-5.3.16-200.fc30.x86_64-x86_64-with-fedora-30-Thirty
19:26:05.122 root INFO Running multibootusb version 9.2.0 from installed system...
19:26:05.124 root INFO Starting multibootusb GUI...
19:26:05.169 root INFO Cleaning old multibootusb directory...
19:26:05.170 root INFO Syslinux exist in multibootusb directory...
19:26:05.170 root INFO Using pyudev for detecting USB drives...
19:26:05.180 root INFO 	/dev/sda
19:26:05.180 root INFO 	/dev/sda1
19:26:05.182 root INFO Selected device /dev/sda
19:26:05.269 root INFO Device /dev/sda is a MBR disk...
19:26:05.308 root INFO USB disk is not mounted and can't update list widget...
19:26:08.731 root INFO No USB disk found...
19:26:08.732 root INFO Using pyudev for detecting USB drives...
19:26:08.751 root INFO 	/dev/sda
19:26:08.753 root INFO 	/dev/sda1
19:26:08.762 root INFO Selected device /dev/sda
19:26:08.853 root INFO Device /dev/sda is a MBR disk...
19:26:08.899 root INFO USB disk is not mounted and can't update list widget...
19:26:11.561 root INFO Selected device /dev/sda1
19:26:19.580 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/arcolinux-v19.12.17.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.cfg" -r 2> /dev/null
19:26:19.600 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/arcolinux-v19.12.17.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.txt" -r 2> /dev/null
19:26:19.607 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/arcolinux-v19.12.17.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*isolinux.bin" -r 2> /dev/null
19:26:19.615 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/arcolinux-v19.12.17.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.lst" -r 2> /dev/null
19:26:19.667 root INFO Persistence support is not available for arcolinux-v19.12.17.iso
19:28:18.073 root INFO Could not find resource 'data/tools/gdisk/gdisk.exe'.
19:28:18.106 root INFO Using python version 3.7.5 (default, Oct 17 2019, 12:09:47) 
[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)]on platform Linux-5.3.16-200.fc30.x86_64-x86_64-with-fedora-30-Thirty
19:28:18.106 root INFO Running multibootusb version 9.2.0 from installed system...
19:28:18.107 root INFO Starting multibootusb GUI...
19:28:18.157 root INFO Cleaning old multibootusb directory...
19:28:18.157 root INFO Syslinux exist in multibootusb directory...
19:28:18.157 root INFO Using pyudev for detecting USB drives...
19:28:18.167 root INFO 	/dev/sda
19:28:18.168 root INFO 	/dev/sda1
19:28:18.170 root INFO Selected device /dev/sda
19:28:18.199 root INFO Device /dev/sda is a MBR disk...
19:28:18.212 root INFO USB disk is not mounted and can't update list widget...
19:33:23.016 root INFO Could not find resource 'data/tools/gdisk/gdisk.exe'.
19:33:23.050 root INFO Using python version 3.7.5 (default, Oct 17 2019, 12:09:47) 
[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)]on platform Linux-5.3.16-200.fc30.x86_64-x86_64-with-fedora-30-Thirty
19:33:23.050 root INFO Running multibootusb version 9.2.0 from installed system...
19:33:23.052 root INFO Starting multibootusb GUI...
19:33:23.094 root INFO Cleaning old multibootusb directory...
19:33:23.095 root INFO Syslinux exist in multibootusb directory...
19:33:23.095 root INFO Using pyudev for detecting USB drives...
19:33:23.104 root INFO 	/dev/sda
19:33:23.104 root INFO 	/dev/sda1
19:33:23.106 root INFO Selected device /dev/sda
19:33:23.134 root INFO Device /dev/sda is a MBR disk...
19:33:23.146 root INFO USB disk is not mounted and can't update list widget...
19:33:30.469 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/arcolinux-v19.12.17.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.cfg" -r 2> /dev/null
19:33:30.498 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/arcolinux-v19.12.17.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.txt" -r 2> /dev/null
19:33:30.526 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/arcolinux-v19.12.17.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*isolinux.bin" -r 2> /dev/null
19:33:30.554 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/arcolinux-v19.12.17.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.lst" -r 2> /dev/null
19:33:30.705 root INFO Persistence support is not available for arcolinux-v19.12.17.iso
19:33:38.007 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/manjaro-xfce-18.1.4-191210-linux54.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.cfg" -r 2> /dev/null
19:33:38.022 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/manjaro-xfce-18.1.4-191210-linux54.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.txt" -r 2> /dev/null
19:33:38.029 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/manjaro-xfce-18.1.4-191210-linux54.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*isolinux.bin" -r 2> /dev/null
19:33:38.037 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/manjaro-xfce-18.1.4-191210-linux54.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.lst" -r 2> /dev/null
19:33:38.064 root INFO Persistence support is not available for manjaro-xfce-18.1.4-191210-linux54.iso
19:33:43.145 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/pop-os_19.10_amd64_nvidia_11.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.cfg" -r 2> /dev/null
19:33:43.158 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/pop-os_19.10_amd64_nvidia_11.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.txt" -r 2> /dev/null
19:33:43.166 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/pop-os_19.10_amd64_nvidia_11.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*isolinux.bin" -r 2> /dev/null
19:33:43.173 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/pop-os_19.10_amd64_nvidia_11.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.lst" -r 2> /dev/null
19:33:43.259 root INFO Persistence option is available.
19:33:43.259 root INFO Persistence Max Size: Unknown
19:33:53.944 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/ubuntu-18.10-server-amd64.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.cfg" -r 2> /dev/null
19:33:54.035 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/ubuntu-18.10-server-amd64.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.txt" -r 2> /dev/null
19:33:54.122 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/ubuntu-18.10-server-amd64.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*isolinux.bin" -r 2> /dev/null
19:33:54.201 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/ubuntu-18.10-server-amd64.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.lst" -r 2> /dev/null
19:33:54.686 root INFO Persistence support is not available for ubuntu-18.10-server-amd64.iso
19:33:58.493 root INFO File not selected...
19:37:59.177 root INFO Could not find resource 'data/tools/gdisk/gdisk.exe'.
19:37:59.221 root INFO Using python version 3.7.5 (default, Oct 17 2019, 12:09:47) 
[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)]on platform Linux-5.3.16-200.fc30.x86_64-x86_64-with-fedora-30-Thirty
19:37:59.221 root INFO Running multibootusb version 9.2.0 from installed system...
19:37:59.223 root INFO Starting multibootusb GUI...
19:37:59.268 root INFO Cleaning old multibootusb directory...
19:37:59.269 root INFO Syslinux exist in multibootusb directory...
19:37:59.269 root INFO Using pyudev for detecting USB drives...
19:37:59.278 root INFO 	/dev/sda
19:37:59.279 root INFO 	/dev/sda1
19:37:59.280 root INFO Selected device /dev/sda
19:37:59.309 root INFO Device /dev/sda is a MBR disk...
19:37:59.322 root INFO USB disk is not mounted and can't update list widget...
19:45:17.663 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/pop-os_19.10_amd64_nvidia_11.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.cfg" -r 2> /dev/null
19:45:17.673 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/pop-os_19.10_amd64_nvidia_11.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.txt" -r 2> /dev/null
19:45:17.680 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/pop-os_19.10_amd64_nvidia_11.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*isolinux.bin" -r 2> /dev/null
19:45:17.689 root INFO Executing ==> 7z -ssc-  x -y "/home/ddean/Downloads/pop-os_19.10_amd64_nvidia_11.iso" -o"/root/.multibootusb/iso_cfg_ext_dir" "*.lst" -r 2> /dev/null
19:45:17.834 root INFO Persistence option is available.
19:45:17.834 root INFO Persistence Max Size: Unknown
19:45:19.120 root INFO Selected USB is a disk. Please select a disk partition from the drop down list
19:45:27.921 root INFO Selected device /dev/sda1
```
